news/_posts/translation/2017-03-10-do-you-speak-prestashop-feb-2017.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – February 2017 edition"
+subtitle: "Translation of 1.7.1 is in full swing!"
+date:   2017-03-13 06:09:47
+authors: [ alexeven ]
+icon: icon-world
+tags: [translation, i18n, l10n]
+---
+
+With 1.7.1 version soon to be released, there has been some movements on the [PrestaShop translation project](https://crowdin.com/project/prestashop-official) last month. And that comes with some good news for translators too!
+
+## Project news
+
+
+### Translation for 1.7.1 are there
+
+The next minor version of PrestaShop, [V.1.7.1, is open for testing as a beta version](http://build.prestashop.com/news/prestashop-1-7-1-0-beta-1/). At the same time, [we opened it for translation on Crowdin](http://build.prestashop.com/news/171-Translations-update/).
+
+We now have a 1.7.1 project, with fresh new strings to translate!
+
+
+### Daily language packs update
+
+Good news! You've been waiting for it, since the 1st of March, both *1.6 and 1.7 translation packs are be updated every day*, automatically.
```

Comment:
"are be updated".

![arr-pirate-jokes](https://cloud.githubusercontent.com/assets/2833749/23857365/57aaa100-07fd-11e7-889b-30fd9cf25ed8.jpg)